<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kamer_portals</class>
 <widget class="QWidget" name="kamer_portals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Portal</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>98</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>portals's kamer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="knop_nw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Richel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="knop_n">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Oliver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="knop_o">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Mees</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>331</width>
     <height>721</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="djog_nanos_2018.qrc">:/plaatjes/heel erg toppie.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Teleporter Rohan</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>knop_n</zorder>
  <zorder>knop_nw</zorder>
  <zorder>knop_o</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources>
  <include location="djog_nanos_2018.qrc"/>
 </resources>
 <connections/>
</ui>
